src/pages/Bridge/IssueForm/index.tsx

```diff
@@ -79,6 +79,8 @@ if (process.env.REACT_APP_RELAY_CHAIN_NAME === POLKADOT) {
 } else {
   throw new Error('Something went wrong!');
 }
+const extraRequiredCollateralTokenAmount =
+  newMonetaryAmount(EXTRA_REQUIRED_COLLATERAL_TOKEN_AMOUNT, GOVERNANCE_TOKEN, true);
```

Comment:
Currently we are using hardcoded `EXTRA_REQUIRED_COLLATERAL_TOKEN_AMOUNT`. Maybe via another ticket, we could update it using some API call.